src/app/app.component.html

```diff
@@ -1,9 +1,9 @@
 <prx-auth [host]="authHost" [client]="authClient"></prx-auth>
 <prx-header>
   <prx-navitem *ngIf="loggedIn" route="/" text="Home"></prx-navitem>
-  <prx-navitem *ngIf="loggedIn" route="/series/new" text="New Series"></prx-navitem>
-  <prx-navitem *ngIf="loggedIn" route="/search" text="Search"></prx-navitem>
-  <prx-navuser *ngIf="loggedIn" [userName]="userName">
+  <prx-navitem *ngIf="authorized" route="/series/new" text="New Series"></prx-navitem>
+  <prx-navitem *ngIf="authorized" route="/search" text="Search"></prx-navitem>
+  <prx-navuser *ngIf="loggedIn" [userinfo]="userinfo">
```

Comment:
Seeing this for reals now ... I'm noting that my display went from "Ryan Cavis" to "ryancavis".  Probably because my preferred_username is `ryancavis`?  That sure sounds like the right thing to do, given that column name.  But if everyone has their lowercase-username in that column, I guess I'd prefer to use the id `name:` over `preferred_username`.  Principle of least surprise!

Reply:
I'm also not a huge fan of how the dropdown is unnecessarily wider than my username.  There seems to be some sort of min-width 200px on it?  Causing that padding on the left/right sides.

![image](https://user-images.githubusercontent.com/1410587/38828362-0bcef69a-4173-11e8-88e6-d23fb87bdf19.png)

Reply:
I think I got the `min-width` from how the corporate site widget works. I could get rid of it altogether (which I suspect might do weird things), or make it smaller, like `150`. Thoughts?

Reply:
Seems like you got rid of it altogether!  Which is fine by me, since that list will always contain `PRX Exchange` for every user.